ta/bti_test/sub.mk

```
@@ -0,0 +1,5 @@
+global-incdirs-y += include
+
+srcs-y += ta_entry.c
+srcs-$(CFG_TA_BTI) += bti_stubs_a64.S
```

This assumes that `CFG_TA_BTI` can only be enabled for 64-bit TAs, but I think it's not the case (does this pass `make check COMPILE_S_USER=32`?).

---

You are right.  

How should I handle this ?

---

Testing `$(sm)` should work:
```suggestion
ifeq ($(sm),ta_arm64)
srcs-$(CFG_TA_BTI) += bti_stubs_a64.S
endif
```

---

Fixed